libbs/api/decompiler_interface.py

```diff
@@ -94,13 +95,23 @@ def __init__(
         self.global_vars = ArtifactDict(GlobalVariable, self, error_on_duplicate=error_on_artifact_duplicates)
 
         self._decompiler_available = decompiler_available
+
+        self._init_config()
+
         if not self.headless:
             args = gui_init_args or []
             kwargs = gui_init_kwargs or {}
             self._init_gui_components(*args, **kwargs)
         else:
             self._init_headless_components()
 
+
+
+   # Checks for config file and creates/loads from it
+    def _init_config(self):
+        config = LibbsConfig.update_or_make()
+        self.config = config
```

Comment:
Class properties used but not defined in the `__init__` can be dangerous. Always define the variable in `__init__` as well: `self.config: Optional[LibbsConfig] = None`.